com.hoptodesk.HopToDesk.json

```diff
@@ -0,0 +1,101 @@
+{
+  "id" : "com.hoptodesk.HopToDesk",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "23.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "command" : "hoptodesk",
+  "modules" : [
+    "shared-modules/libappindicator/libappindicator-gtk3-12.10.json",
+    {
+      "name": "libappindicator"
+    },
+    {
+      "name": "xdotool",
+      "no-autogen": true,
+      "make-args": [
+          "WITHOUT_RPATH_FIX=1"
+      ],
+      "make-install-args": [
+          "PREFIX=${FLATPAK_DEST}",
+          "INSTALLMAN=${FLATPAK_DEST}/share/man"
+      ],
+      "sources": [
+          {
+              "type": "archive",
+              "url": "https://github.com/jordansissel/xdotool/archive/v3.20211022.1/xdotool-3.20211022.1.tar.gz",
+              "sha256": "82b15a944a5e82fee15e0f6116bd9f642bc3d0bb6989fc0ca5ad9dfe35de0847",
+              "x-checker-data": {
+                  "type": "anitya",
+                  "project-id": 8648,
+                  "stable-only": true,
+                  "url-template": "https://github.com/jordansissel/xdotool/archive/v$version/xdotool-$version.tar.gz"
+              }
+          }
+      ],
+      "cleanup": [
+          "/share/man"
+      ]
+    },
+    {
+      "name" : "libpam",
+      "buildsystem" : "autotools",
+      "sources" : [
+        {
+          "url" : "https://github.com/linux-pam/linux-pam/releases/download/v1.6.1/Linux-PAM-1.6.1.tar.xz",
+          "sha256" : "f8923c740159052d719dbfc2a2f81942d68dd34fcaf61c706a02c9b80feeef8e",
+          "type" : "archive"
+        }
+      ]
+    },
+    {
+      "name" : "HopToDesk",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "url": "https://www.hoptodesk.com/hoptodesk.deb",
```

Comment:
the .deb isn't versioned? This will be great to have the package not build at random times.

Reply:
also why isn't it built from source given that licensing isn't a problem. This would also allow supporting aarch64 at the same time.